app/assets/scripts/map.js

```diff
@@ -14,7 +15,7 @@ var infowindow;
 function initMap() {
   map = new google.maps.Map(document.getElementById('map'), {
     center: {lat: 39.949, lng: -75.181}, // TODO(#52): Do not hardcode this.
-    zoom: 13
+    zoom: focusZoom,
```

Comment:
This also probably shouldn't be hard-coded. I suppose it can be hard-coded here, but the map should dynamically update the zoom depending on the number of markers.